front-page.php

```diff
@@ -34,11 +34,12 @@
 
 		</main><!-- #main -->
 	</div><!-- #primary -->
-<?php get_template_part( 'template-parts/content', 'secondary' ) ?>	
+<?php get_template_part( 'template-parts/content', 'secondary' ) ?>
 <?php get_sidebar(); ?>
 </div><!-- .o-layout -->
 
 <?php get_template_part( 'template-parts/acf', 'links' );?>
+<?php get_template_part( 'template-parts/acf', 'featured-pages' );?>
```

Comment:
Should we rename this featured-content to make it more generic?

Reply:
Agreed and done.